addons/website/static/src/interactions/carousel_slider.edit.js

```diff
@@ -0,0 +1,65 @@
+import { registry } from "@web/core/registry";
+import { Interaction } from "./carousel_slider";
+
+export class CarouselSliderEdit extends CarouselSlider {
+    // Pause carousel in edit mode.
+    carouselOptions = {ride: false, pause: true};
+
+    setup() {
+        Object.assign(this.dynamicContent, {
+            "_root": {
+                "t-on-contentChanged": this.onContentChanged,
+            },
+        });
+        // Only for carousels having the `Carousel` and `CarouselItem` options
+        // (i.e. matching the `section > .carousel` selector).
+        if (this.el.matches("section > .carousel")) {
+            // TODO Handle translation mode.
+            // && !this.options.wysiwyg.options.enableTranslation) {
+            Object.assign(this.dynamicContent, {
+                ".carousel-control-prev, .carousel-control-next": {
+                    "t-att-data-bs-slide": () => undefined,
+                    "t-on-mousedown": this.onControlClick,
+                },
+                ".carousel-indicators > *": {
+                    "t-att-data-bs-slide-to": () => undefined,
+                    "t-on-mousedown": this.onControlClick,
+                },
+            });
+        }
+        super.setup();
+    }
+
+    // TODO See if slide indices to need to be re-generated or is restore
+    // sufficient.
+    /*
+    destroy() {
+        super.destroy();
+        /*
+        if (this.editableMode && this.el.matches("section > .carousel")) {
+            // TODO Handle translation mode.
+//                && !this.options.wysiwyg.options.enableTranslation) {
+            // Restore the carousel controls.
+            const indicatorEls = this.el.querySelectorAll(".carousel-indicators > *");
+            this.options.wysiwyg.odooEditor.observerUnactive("restore_controls");
+            indicatorEls.forEach((indicatorEl, i) => indicatorEl.setAttribute("data-bs-slide-to", i));
+            this.options.wysiwyg.odooEditor.observerActive("restore_controls");
+        }
+    }
+    */
+
+    onContentChanged() {
+        this.computeMaxHeight();
+    }
+
+    /**
+     * Redirects a carousel control click on the active slide.
+     */
+    onControlClick() {
```

Comment:
this one is not necessary, it's in the CarouselSlider class

Reply:
Oops it shouldn't. Thanks